app/assets/javascripts/components/high_order/editable.cjsx

```diff
@@ -28,13 +28,14 @@ Editable = (Component, Stores, Save, GetState, Label) ->
       if permissions && @state.editable
         unless save_only
           className = 'controls'
-          cancel = (
-            <button onClick={@cancelChanges} className='button'>{I18n.t('editable.cancel')}</button>
-          )
+          unless SaveOnly
```

Comment:
@ragesoss I'm not 100% in touch with what's happening here, but the nested `unless` blocks and the two different "Save Onlys" makes this area a little tough to decipher. Perhaps adding some comments at least might help us refactor in the future?